From b60115a1c00e30e15395ca0c4bcaf22c3ac431cb Mon Sep 17 00:00:00 2001
Message-Id: <b60115a1c00e30e15395ca0c4bcaf22c3ac431cb.1536572113.git.tredaelli@redhat.com>
From: Timothy Redaelli <tredaelli@redhat.com>
Date: Tue, 31 Jul 2018 15:15:27 +0200
Subject: [PATCH 1/2] net/mlx4: avoid stripping the glue library

Stripping binaries at build time is usually a bad thing since it makes
impossible to generate (split) debug symbols and this can lead to a more
difficult debugging.

Fixes: 27cea11686ff ("net/mlx4: spawn rdma-core dependency plug-in")
Cc: stable@dpdk.org

Signed-off-by: Timothy Redaelli <tredaelli@redhat.com>
Acked-by: Luca Boccassi <bluca@debian.org>
Acked-by: Christian Ehrhardt <christian.ehrhardt@canonical.com>
(cherry picked from commit d7a4e99d84f961c72eca7541e090cc3f43b60fb8)
---
 drivers/net/mlx4/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/mlx4/Makefile b/drivers/net/mlx4/Makefile
index d24565cd1..6228520c2 100644
--- a/drivers/net/mlx4/Makefile
+++ b/drivers/net/mlx4/Makefile
@@ -139,7 +139,7 @@ $(LIB): $(LIB_GLUE)
 $(LIB_GLUE): mlx4_glue.o
 	$Q $(LD) $(LDFLAGS) $(EXTRA_LDFLAGS) \
 		-Wl,-h,$(LIB_GLUE) \
-		-s -shared -o $@ $< -libverbs -lmlx4
+		-shared -o $@ $< -libverbs -lmlx4
 
 mlx4_glue.o: mlx4_autoconf.h
 
-- 
2.17.1

